docroot/themes/humsci/humsci_basic/src/scss/components/_site-search.scss

```diff
@@ -10,37 +9,46 @@
     @include hide-visually;
   }
 
+  .form-actions {
+    position: relative;
+  }
+
   input {
     @include font-smoothing;
-    @include padding(0.6rem 2rem);
+    padding: hb-calculate-rems(12px) hb-calculate-rems(36px) hb-calculate-rems(10px) hb-calculate-rems(14px);
 
     display: inline-block;
-    height: 4rem;
+    height: hb-calculate-rems(38px);
     max-width: 100%;
-    border-radius: 3rem;
-    font-size: 1.6rem;
+    border-radius: hb-calculate-rems(4px);
+    font-size: hb-calculate-rems(16px);
 
     &::placeholder {
-      color: $su-color-black;
+      color: $hb-color--black;
       opacity: 1; // For Firefox
+
+      @include hb-colorful {
+        color: darken($su-color-driftwood, 35%);
```

Comment:
for the driftwood color, you dont have to use the hb-colorful theme mixin if you dont want!

Reply:
Good to know! So driftwood is going to be consistent throughout all the themes then?

Reply:
yes! it's a part of the stanford identity colors